UI/data/locale/en-US.ini

```diff
@@ -250,6 +250,10 @@ Updater.Running.Title="Program currently active"
 Updater.Running.Text="Outputs are currently active, please shut down any active outputs before attempting to update"
 Updater.NoUpdatesAvailable.Title="No updates available"
 Updater.NoUpdatesAvailable.Text="No updates are currently available"
+Updater.RepairButUpdatesAvailable.Title="Updates available"
+Updater.RepairButUpdatesAvailable.Text="Repairing is only possible for the latest version available. Please use the regular updating process to update and repair your installation."
```

Comment:
Perhaps instead of "regular updating process" we could say "Help / Check For Updates" to make it more explicit.